examples/create-mcs-db-cluster-with-shards/provider.tf

```diff
@@ -0,0 +1,15 @@
+provider "mcs" {
+    username          = "user@mail.ru"
+    password          = "password"
+    project_id        = "project_id"
+    auth_url          = "https://infra.mail.ru/identity/v3/"
```

Comment:
I've checked, we support default value in code and it's the same as provided.
I think it's better not to add it to examples just not to disturb clients here.
Please remove it here and below.

I'm removing them from old examples as well.
https://github.com/MailRuCloudSolutions/terraform-provider-mcs/pull/21

We'll add the description about this optional value to docs then (you don't need to do this).

Reply:
Fixed

Reply:
thx